.github/workflows/publish.yml

```
@@ -0,0 +1,132 @@
+      contents: write
+      pull-requests: read
+    needs: make-changelog
+    steps:
+      - name: Create Draft Release
+        id: create_release
+        uses: softprops/action-gh-release@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: ${{ github.ref_name }}
+          body: |
```

actionlint

property "draft-changelog" is not defined in object type {make-changelog: {outputs: {release_body: string}; result: string}}

(expression)